data/drak/indigenous.txt

```diff
@@ -18,18 +18,20 @@ ship "Subsidurial"
 		"hull" 11600
 		"automaton" 1
 		"mass" 700
+		"cargo space" 70
 		"drag" 8.2
 		"heat dissipation" .8
-		"cargo space" 70
+		"fuel capacity" 100
+		"outfit space" 20
 		"thrust" 10
 		"turn" 800
 		"hull repair rate" 3
 		"energy capacity" 1
-		"fuel capacity" 100
 		"inscrutable" 1
 		"gaslining" 1
 	outfits
 		Mouthparts? 3
+		Hyperdrive
```

Comment:
You could just give the ship a `"hyperdrive" 1` attribute instead of needing to give it the outfit. (Would still need the fuel capacity, though.)

Reply:
Good suggestion. Made the change.